pixeltable/catalog/catalog.py

```diff
@@ -210,12 +218,28 @@ def __init__(self) -> None:
         self._in_write_xact = False
         self._x_locked_tbl_ids = set()
         self._modified_tvs = set()
+        self._roll_forward_ids = set()
         self._undo_actions = []
         self._in_retry_loop = False
         self._column_dependencies = {}
         self._column_dependents = None
         self._init_store()
 
+    def _active_tbl_clause(
```

Comment:
Would benefit from a docstring with a brief description of what it's doing.

Reply:
Done.